V0.5.2BeBuilder/bebuilder.rdef

```diff
@@ -0,0 +1,70 @@
+
+resource app_signature "application/x-vnd.bebuilder";
+
+resource app_version {
+	major  = 0,
+	middle = 0,
+	minor  = 0,
```

Comment:
No worries, We will worry about filling in the version info if/when a tagged released is made.